### ISSUE TITLE
Formatting issues with some forums

### ISSUE THREAD

**Issue by rtxa** [rtxa/mdtobb#1]
# Problem

Depending of how the forum renders BBCode, some headings will not have any padding or margin, requiring you to add an extra blank line to look properly. The same things happens with paragraphs and lists.

# Origin of the issue

This is mostly an issue with the theme of the forum for using some bad CSS practices, or the forum itself not translating properly the BBCode to HTML.

# Temporary fix

To circumvent this, I added the `Add new line` option for the next elements:

- Header
- Paragraph
- List

So basically isn't enough with some forums to choose an style and convert, you also need to use more customizing options. Luckily, you can save these styles and use them later.

-----------------------

I'm gonna leave open this issue by the moment until we can find a proper fix for this issue. Be free to add any suggestions about how should I handle this type of forums, if you require to add more customization options to fix the formatting, etc.
